tests/BoostOptionTypeTest.cpp

```diff
@@ -13,17 +13,17 @@
 #include <boost/container/static_vector.hpp>
 #include <boost/container/vector.hpp>
 #include <string>
+#include <tuple>
+#include <utility>
 #include <vector>
 
-using namespace boost::container;
```

Comment:
:heart: (change not related to this PR's topic, but like the change)

Reply:
IWYU was confusing slist from boost with the one from gcc, obviously couldn't include that one, so that just triggered getting rid of the boost container namespace and using it more directly with the full name of the types